.github/workflows/scripts/health_check_no.sh

```diff
@@ -0,0 +1,24 @@
+# .github/workflows/scripts/health_check_no.sh
+#!/bin/bash
+
+port="$1"
```

Comment:
Add comments above to fully describe what the args do and the logic of the script for future maintainability